fuzz_test.go

```diff
@@ -0,0 +1,27 @@
+// SPDX-FileCopyrightText: 2023 The Pion community <https://pion.ly>
+// SPDX-License-Identifier: MIT
+package dtls
+
+import (
+	"os"
+	"testing"
+)
+
+func FuzzUnmarshalBinary(f *testing.F) {
+	TestResumeClient, err := os.ReadFile("testdata/seed/TestResumeClient.raw")
```

Comment:
Please have a look at the [Go fuzzing tutorial](https://go.dev/doc/tutorial/fuzz)

The Go fuzzing enging has built-in support for reading files from the `testdata` directory.

Reply:
Oh, My bad https://go.dev/security/fuzz/#glos-seed-corpus
```txt
and the files in the testdata/fuzz/{FuzzTestName} directory within the package
```

Reply:
> Oh, My bad https://go.dev/security/fuzz/#glos-seed-corpus
> 
> ```
> and the files in the testdata/fuzz/{FuzzTestName} directory within the package
> ```

For a minute, I thought I was wrong.
Anything within `testdata/fuzz/{FuzzTestName}` that crashes cannot be accepted as a seed for fuzzing. So, yes, my previous `commit`  was on point.